document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/host-os.md

---
path: "/docs/create-installer/host-preflights/operating-system"
date: "2022-01-13"
weight: 30
linktitle: "Operating System"
title: "Operating System"
---
 

> The operating system host preflight check can be used to detect and validate the name and version of the OS installed on the machine.

Reviewer: ```suggestion
You can use the operating system host preflight check to detect and validate the name and version of the OS installed on the machine.
```
